common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -763,7 +763,15 @@ public static boolean supportExportDefaultJavaSdk(@NotNull final String pantsExe
     }
 
     final String defaultPlatform = exportResult.getJvmPlatforms().getDefaultPlatform();
-    final String jdkName = String.format("JDK from pants %s", defaultPlatform);
+
+    // if defaultPlatform is java8, then jdkName should be 1.8_from_pants
+    String jdkName = "1.x_from_pants";
+    for (String version : ContainerUtil.newArrayList("6", "7", "8", "9")) {
+      if (defaultPlatform.contains(version)) {
```

Comment:
Switched to using IJ's framework to detect jdk version. so jdkInfo.getVersion() returns `java version "1.8.0_121"` then we can check for `1.7` `1.8` `1.9`